modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/ActivationKeysLayout/Inputs/index.jsx

```diff
@@ -117,6 +118,16 @@ const ActivationKeysInputs = ({
 		}
 	}, [selectDateInterval, selectedAccountSubscriptionName]);
 
+	const productNameData = [productTitle, selectedAccountSubscriptionName];
+	const productName = productNameData.map((product) =>
+		product.replaceAll(/[^a-zA-Z0-9]/g, '').toLowerCase()
+	);
+	const productNameFormatted = productName.join('');
+
+	const projectNameFormatted = `${projectName
+		.replaceAll(/[^a-zA-Z0-9]/g, '')
+		.toLowerCase()}`;
+
```

Comment:
Since you need this data inside of handleClick, remove from here.

- This will avoid to re-computate those validations for each render

Reply:
Thank you, done.